resty

```diff
@@ -168,6 +170,17 @@ for my $var (sort keys %ENV) {
     $env_list .= "env $var;\n";
 }
 
+my $main_include_directive = '';
+my $http_include_directive = '';
+
+if ($main_include) {
+    $main_include_directive = "include $main_include;"
```

Comment:
Good point about the relative paths. Intuitively it feels heavy to start doing globbing ourselves, but it could be done.  I can think of three different ways of doing it:

- Full blown globbing as discussed above
- We keep it simple and require the arguments to be one file, expand them to absolute paths if necessary and verify that they exist
- We treat it as a somewhat opaque path reference which we expand to an absolute reference if needed, keeping any wildcards for nginx to handle

I'm leaning towards the second alternative as that seems simplest and most reliable. Other alternatives could be implemented later if and when they're requested.